### PR TITLE
Fixes documentation error in compute_router_nat

### LINKED ISSUE #3516: Documentation: r/google_compute_router_nat example does not include required subnetwork.source_ip_ranges_to_nat field
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment
* If an issue is assigned to the "modular-magician" user, it is either in the process of being autogenerated, or is planned to be autogenerated soon. If an issue is assigned to a user, that user is claiming responsibility for the issue. If an issue is assigned to "hashibot", a community member has claimed the issue already.

### Terraform Version

master @b21921fad297a983937286d63b9b0dab08ba3b1a

### Affected Resource(s)

* compute_router_nat

### Description

This is a documentation error with the following file: [https://github.com/kkleidal/terraform-provider-google/blob/b21921fad297a983937286d63b9b0dab08ba3b1a/website/docs/r/compute_router_nat.html.markdown](https://github.com/kkleidal/terraform-provider-google/blob/b21921fad297a983937286d63b9b0dab08ba3b1a/website/docs/r/compute_router_nat.html.markdown).

The second example contains the subnetwork block:
```hcl
subnetwork {
  name = "${google_compute_subnetwork.subnetwork.self_link}"
}
```
which doesn't contain the required argument `source_ip_ranges_to_nat`.

### PR DESCRIPTION
**Pull request by kkleidal**
Fixes missing required argument in second example of compute_router_nat.

Fixes #3516

### CODE REVIEW

File: website/docs/r/compute_router_nat.html.markdown [hunk 1]
```diff
@@ -87,6 +87,7 @@ resource "google_compute_router_nat" "advanced-nat" {
   source_subnetwork_ip_ranges_to_nat = "LIST_OF_SUBNETWORKS"
   subnetwork {
     name = "${google_compute_subnetwork.subnetwork.self_link}"
```
Comment (rileykarson) [inline website/docs/r/compute_router_nat.html.markdown]: Do you mind bumping the `=` in this line up so it lines up with the argument below?

Comment (kkleidal) [reply website/docs/r/compute_router_nat.html.markdown]: @rileykarson : fixed it and squashed down to a single commit (8816d3a)

Comment (rileykarson) [reply website/docs/r/compute_router_nat.html.markdown]: Thanks! I'll upstream this change to Magic Modules so it hits `google-beta` too, and merge this PR.

**Comment by rileykarson** [comment]
Thanks again @kkleidal!

**Pull request merged**